sdk/python/kfp/v2/components/__init__.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from kfp.v2.components.component_factory import create_component_from_func
```

Comment:
Sorry, we actually don't want to include this in v2. This will not create a "v2 experience". We expect users to always use `@component` instead.